### ISSUE TITLE
My test of FID score is much higher than author's

### ISSUE THREAD

**Issue by YijiFeng** [GaParmar/img2img-turbo#92]
Hi, @GaParmar . I use the following codes to calculate the fid score, but the result is much higher than the value of wandb curve. 
This is my code:
`from cleanfid import fid

real_dir = 'data/plant/test_A'
fake_dir = 'output/cyclegan_turbo/plant/fid-15001/samples_a2b'
score = fid.compute_fid(real_dir,fake_dir,mode='clean')
print(f'fid score:{score}')`
The result:
![image](https://github.com/user-attachments/assets/1c717468-071e-4c45-a63a-cab8388cec84)
The value of the wandb curve:
![image](https://github.com/user-attachments/assets/51074292-f255-4832-9a47-c9dd392a6bc0)
Why does this happen?

**Comment by GaParmar** [comment]
Maybe this is because of the number of images used for computing the score. 
Are you using the same number of images during the training and testing? 

-Gaurav

**Comment by YijiFeng** [comment]
> Maybe this is because of the number of images used for computing the score. Are you using the same number of images during the training and testing?
> 
> -Gaurav

Yes, I used the different number of images during the training and testing, thanks a lot. I will test it with the same.
